src/sql/workbench/browser/parts/views/customView.ts

```diff
@@ -700,6 +701,9 @@ class TreeDataSource implements IAsyncDataSource<ITreeItem, ITreeItem> {
 				// So in order to enable this we need to tell the tree to refresh this node so it will ask us for the data again
 				setTimeout(() => {
 					this.treeView.collapse(node);
+					if (e.code === UserCancelledConnectionError.code()) {
```

Comment:
You should be able to do

`e instanceof UserCancelledConnectionError`

If UserCancelledConnectionError was an interface that is indeed lost, but actual classes are supported in Javascript so you're able to do the checks just fine https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes

Reply:
Yeah you're right. For some reason I thought I was making an interface.